proto/proto/penumbra/core/stake/v1alpha1/stake.proto

```diff
@@ -146,16 +146,14 @@ message Undelegate {
   crypto.v1alpha1.IdentityKey validator_identity = 1;
   // The index of the epoch in which this undelegation was performed.
   uint64 start_epoch_index = 2;
-  // The index of the epoch in which unbonding should complete.
-  uint64 end_epoch_index = 3;
   // The amount to undelegate, in units of unbonding tokens.
-  crypto.v1alpha1.Amount unbonded_amount = 4;
+  crypto.v1alpha1.Amount unbonded_amount = 3;
```

Comment:
It's not worth undoing, but in general it's better to leave the other field numbers unchanged, and just leave a gap, because that way parsing can be backwards/forwards compatible at the data interchange level.